x-pack/plugins/apm/public/components/app/service_inventory/service_inventory.stories.tsx

```diff
@@ -61,7 +60,7 @@ const stories: Meta<{}> = {
             <MockUrlParamsContextProvider
               params={{
                 comparisonEnabled: true,
-                comparisonType: TimeRangeComparisonEnum.DayBefore,
+                offset: '1d',
```

Comment:
should this not be -1d? I realise it's test data, but instinctively I would say day before should be offset: -1d, and if we use 1d here it might confuse some folks as to how it's interpreted.

Reply:
See https://github.com/elastic/kibana/pull/127812#pullrequestreview-915757409